snippets/product/type/with-variations/product.liquid

```diff
@@ -0,0 +1,72 @@
+<div class="row">
+    <div class="col-xs-12">
+        <!-- <a href="#">{{ product.properties['Brand'].value }}</a> -->
+        <h1>{{ product.title }}</h1>
+        <ul class="list-inline">
+            <li>
+                <span class="text-muted">{{ 'products.product.sku' | t }}</span>&nbsp;<strong itemprop="productID">{{ product.sku }}</strong>
+            </li>
+            <li>
+                <span class="text-muted">{{ 'products.product.mmn' | t }}</span>&nbsp;<strong itemprop="model">{{ product.properties['MMN'].value }}</strong>
+            </li>
+            <li>
+                <span class="text-muted">UNSPSC</span>&nbsp;<strong>#41112701</strong>
+            </li>
+        </ul>
+    </div>
+</div>
+<div class="row flexible">
+    <div class="col-md-3">
+        {% include 'product/image' %}
+        {% include 'product/compare' %}
+        {% include 'product/product-compare-list-bar' %}
+    </div>
+
+    <div class="col-md-6">
+        {% if product.properties.size > 0 %}
+            {% include 'product/type/common/product-properties' %}
+            {% include 'product/type/common/product-description' %}
+        {% endif %}
+        {% include 'product/variations-panel' %}
+    </div>
+
+    <div class="col-md-3" ng-controller="priceController">
+        <div class="panel panel-default price-panel">
+            <div class="panel-heading">
+                <div class="row">
+                    <div class="col-md-12">
+                        <span class="h4">Price & delivery</span>
+                    </div>
+                </div>
+            </div>
+            <div class="panel-body">
+                <div class="price-block">
+                  {% assign product = product %}
+                  {% include 'product/price' %}
+                </div>
+
+                <div class="delivery-cart">
+                    <div ng-controller="purchaseController" ng-init='loadPaymentPlan({{ settings.available_paymentPlans | json }}, "product", "{{ product.id }}")'>
+                        <p ng-show="shipmentType == 'shipping' && shipmentAddress">Expected to arrive: <strong ng-bind="availability['{{ product.id }}'].expectedArrival | date: 'fullDate'"></strong></p>
+                        <p ng-show="shipmentType == 'pickup' && shipmentFulfillmentCenter">Available for pickup: <strong ng-bind="availability['{{ product.id }}'].availableSince | date: 'fullDate'"></strong></p>
+                        <vc-payment-plan available-payment-plans="::availablePaymentPlans"></vc-payment-plan>
+                        <div class="form-group">
+                            <ng-controller ng-controller="commonProductCardController">
+                                {% include 'product/quantity' %}
+                            </ng-controller>
+                        </div>
+                    </div>
+                </div>
```

Comment:
This part looks exactly the same as for other types. May we reuse the code?

Reply:
done